corehq/apps/sso/certificates.py

```diff
@@ -11,43 +13,59 @@
 
 
 def create_key_pair():
```

Comment:
Although the changes look okay but are there tests for this file?

Reply:
There are tests that exercise the functions in this file (indirectly). I didn't find any unit tests for this file specifically.